addon/-private/system/model/internal-model.js

```diff
@@ -518,6 +518,8 @@ export default class InternalModel {
 
   setupData(data) {
     heimdall.increment(setupData);
+    this.store._internalModelHasRelationshipData(this.modelName, this.id, data.relationships);
```

Comment:
This sounds like a boolean returning method, maybe something like received or process or something would be better?